components/event-publisher/org.wso2.carbon.event.publisher.core/src/main/java/org/wso2/carbon/event/publisher/core/internal/EventPublisher.java

```diff
@@ -280,4 +309,21 @@ public void process(Event event) {
     public org.wso2.siddhi.query.api.definition.StreamDefinition getStreamDefinition() {
         return streamDefinition;
     }
+
+    public class EventWrapper{
+
+        private Event event;
+        private long timestampInMilies;
```

Comment:
Thanks @nirmal070125, mentioned typos are corrected as "millis" in pull request - https://github.com/wso2/carbon-analytics-common/pull/121